diff -up simutrans-123.0.1/Makefile.64~ simutrans-123.0.1/Makefile
--- simutrans-123.0.1/Makefile.64~	2023-05-11 17:22:51.356618322 +0200
+++ simutrans-123.0.1/Makefile	2023-05-11 17:24:00.769213455 +0200
@@ -37,6 +37,8 @@ ifeq ($(findstring $(OSTYPE), $(OSTYPES)
   $(error Unkown OSTYPE "$(OSTYPE)", must be one of "$(OSTYPES)")
 endif
 
+CFLAGS += -DUSE_HW -DUSE_C
+
 ifeq ($(BACKEND),posix)
   COLOUR_DEPTH := 0
 else
