diff -up simutrans-123.0.1/sys/simsys.cc.homepath~ simutrans-123.0.1/sys/simsys.cc
--- simutrans-123.0.1/sys/simsys.cc.homepath~	2023-05-11 17:22:05.878233813 +0200
+++ simutrans-123.0.1/sys/simsys.cc	2023-05-11 17:22:37.215498277 +0200
@@ -395,7 +395,7 @@ char const *dr_query_homedir()
 #elif defined __ANDROID__
 	tstrncpy(buffer,SDL_GetPrefPath("Simutrans Team","simutrans"),lengthof(buffer));
 #else
-	sprintf(buffer, "%s/simutrans", getenv("HOME"));
+	sprintf(buffer, "%s/.simutrans", getenv("HOME"));
 #endif
 
 	// create directory and subdirectories
