--- simutrans-0.120.0.1/Makefile.64~	2014-12-31 21:14:15.351010543 +0100
+++ simutrans-0.120.0.1/Makefile	2014-12-31 21:15:45.968373661 +0100
@@ -18,6 +18,8 @@ ifeq ($(findstring $(OSTYPE), $(OSTYPES)
   $(error Unkown OSTYPE "$(OSTYPE)", must be one of "$(OSTYPES)")
 endif
 
+CFLAGS += -DUSE_HW -DUSE_C
+
 ifeq ($(OSTYPE),amiga)
   STD_LIBS ?= -lunix -lSDL_mixer -lsmpeg -lvorbisfile -lvorbis -logg
   CFLAGS += -mcrt=newlib -DUSE_C -DSIM_BIG_ENDIAN -gstabs+
