--- simutrans-0.120.0.1/Makefile.omv~	2014-12-31 21:25:05.662661834 +0100
+++ simutrans-0.120.0.1/Makefile	2014-12-31 21:25:17.526085836 +0100
@@ -566,7 +566,7 @@ ifeq ($(BACKEND),opengl)
     SDL_LDFLAGS := $(shell $(SDL_CONFIG) --libs)
   endif
   CFLAGS += $(SDL_CFLAGS)
-  LIBS   += $(SDL_LDFLAGS) -lglew32
+  LIBS   += $(SDL_LDFLAGS) -lGLEW
   ifeq ($(OSTYPE),mingw)
     LIBS += -lopengl32
   else
