--- simutrans-0.120.0.1/simsys.cc.homepath	2014-12-31 21:12:10.373382220 +0100
+++ simutrans-0.120.0.1/simsys.cc	2014-12-31 21:12:47.717003040 +0100
@@ -89,7 +89,7 @@ char const* dr_query_homedir()
 #elif defined __APPLE__
 	sprintf(buffer, "%s/Library/Simutrans", getenv("HOME"));
 #else
-	sprintf(buffer, "%s/simutrans", getenv("HOME"));
+	sprintf(buffer, "%s/.simutrans", getenv("HOME"));
 #endif
 
 	dr_mkdir(buffer);
